src/api.ts

```diff
@@ -14,12 +14,13 @@ const TARGET_TYPE_DISPLAY: Record<string, string> = {
 type TypeTargets = Record<string, string[] | undefined>;
 
 export class Targets {
-    private _pageSwitchOptions: [string, number][] = [];
     private activeIndex: number;
-
-    private readonly types: string[];
     private readonly inner: TypeTargets;
-    
+    private readonly types: string[];
+
+    private pageSwitchIndexes: number[] = [];
```

Comment:
Isn't pageSwitchIndexes an array where each member is its own index? i.e. [0,1,2,3,..]?

Reply:
It will be according to what page isn't selected, can be `[0, 2, 3,...]` so I want to sync the indexes of the options and the indexes of the corresponding types

Reply:
Can we remove pageSwitchIndexes and ust use indexOf on pageSwitchOptions? Think we can take the performance it, if that's the reason for it, for the sake of clarity

Reply:
But I want the index of `types` not the `pageSwitchOptions` and the the problem is the formatting of "Show XXXXXXs"